### PR TITLE
Add Wikibase Local Media extension to 1.35 bundle

### PR DESCRIPTION
**Pull request by JeroenDeDauw**

### CODE REVIEW

File: wikibase/1.35/bundle/Dockerfile [hunk 1]
```diff
@@ -13,6 +13,8 @@ bash download-extension.sh WikibaseCirrusSearch;\
 bash download-extension.sh UniversalLanguageSelector;\
 bash download-extension.sh cldr;\
 bash download-extension.sh EntitySchema;\
+RUN git clone https://github.com/ProfessionalWiki/WikibaseLocalMedia --depth=1
```
Comment (addshore) [inline wikibase/1.35/bundle/Dockerfile:16]: Does WikibaseLocalMedia master branch assure compatibility with 1.35 of Mediawiki and Wikibase forever?
Or are we better to pin to a specific commit or other tag etc?

**Review by addshore — CHANGES_REQUESTED** [review]
Open question

File: wikibase/1.35/bundle/Dockerfile (hunk 1 above)
Comment (JeroenDeDauw) [reply wikibase/1.35/bundle/Dockerfile:16]: If in some years MW 1.35 compat is dropped, could we not just update the branch here? Presumably by that point the Docker images will look differently anyway, and perhaps support a composer package version.

Pinning a specific release will lead to more derp in the short and mid term future.

Comment (JeroenDeDauw) [reply wikibase/1.35/bundle/Dockerfile]: I've created a REL1_35 branch. Hope I don't forgetti to update it when there is a new release. Composer support would still be much better.

File: wikibase/1.35/bundle/Dockerfile [hunk 2]
```diff
@@ -13,6 +13,8 @@ bash download-extension.sh WikibaseCirrusSearch;\
 bash download-extension.sh UniversalLanguageSelector;\
 bash download-extension.sh cldr;\
 bash download-extension.sh EntitySchema;\
+RUN git clone https://github.com/ProfessionalWiki/WikibaseLocalMedia --depth=1 --branch=REL1_35
```
Comment (addshore) [inline wikibase/1.35/bundle/Dockerfile:16]: the .git directory should be deleted ideally

**Review by addshore — COMMENTED** [review]
1 last comment then I think we can get this merged (once CI is fixed)

**Review by addshore — APPROVED** [review]
Just need to fix the CI in another PR now

**Comment by manicki** [comment]
I'd like to make one remark for the sake of clarity: Those docker images are considered a prototype by WMDE, and we're currently working on creating the officially supported docker images of Wikibase and related software. We intend to provide a "suite" of software (i.e. including additional extensions and tools), but we have not decided on the list of the software elements that will be included there.

While we will happily see "Wikibase Local Media" extension included in the current prototypical docker images, this does not mean we are committed to include it in the future Docker images or other release formats we're working on.

Mentioning it clear to not, not intentionally, create false expectations regarding WMDE plans.

**Comment by addshore** [comment]
This is included in the new WMDE maintained docker images

**Pull request closed without merging**